lib/htsget-server/docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
         opa_secret: "${CANDIG_OPA_SECRET}"
         opa_url: "${OPA_URL}"
         candig_auth: "${CANDIG_AUTHORIZATION}"
+        minio_url: "${CANDIG_DOMAIN}:${MINIO_PORT}"
```

Comment:
MinIO port will not be accessible in prod. Accessing MinIO will need to go through Tyk

Reply:
This will get deprecated pretty quickly in https://github.com/CanDIG/CanDIGv2/pull/142: after I move S3 stuff (including local Minio) to be accessed through Vault, this isn't needed anymore.